include/cucumber-cpp/internal/RegistrationMacros.hpp

```diff
@@ -21,11 +21,18 @@
 // **************                 CUKE OBJECTS                 ************** //
 // ************************************************************************** //
 
+#if defined(CUKE_ENABLE_BREAKPOINT_SUPPORT)
```

Comment:
Why? 

From my point of view, there's lots of added complexity but no benefit. Am I missing something?